packages/module/src/ConversationAlert/ConversationAlert.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Alert, TextContent } from '@patternfly/react-core';
+
+import { createUseStyles } from 'react-jss';
+
+export interface ConversationAlertProps {
+  title: string;
+  variant?: 'success' | 'danger' | 'warning' | 'info' | 'custom';
```

Comment:
can you make it so the default is `info`? This should be set on the props - see how i did it [here](https://github.com/patternfly/virtual-assistant/blob/main/packages/module/src/VirtualAssistant/VirtualAssistant.tsx#L75).

Also, it would be great if you could add in-line documentation just for completeness see [here](https://github.com/patternfly/virtual-assistant/blob/main/packages/module/src/VirtualAssistant/VirtualAssistant.tsx#L53).